src/main/resources/META-INF/plugin.xml

```diff
@@ -285,6 +288,12 @@
                     <override-text place="GoToAction" text="Cody: Dismiss"/>
                 </action>
 
+                <action id="cody.inlineEditEditCode"
+                        text="Edit Code"
+                        class="com.sourcegraph.cody.edit.InlineEditPromptEditCodeAction" >
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl ENTER" keymap="$default"/>
```

Comment:
I see such comment from @steveyegge in this file:

```
    <!--
        N.B. You have to declare all the Mac keystrokes with ctrl or alt separately,
        even if their definition is identical to $default, because otherwise it will
        result in the actual keybindings switching to command/meta and/or shift.
    -->
```

Could you please check?

Reply:
Also, please add `<override-text place="GoToAction" text="Cody: Confirm Code Edition"/>`
or something along those lines (it is for search everywhere).

Reply:
`text="Cody: Confirm Code Edition"` that would be different from the text in the button and hence confusing imho

Reply:
Problem is that we have two "Edit Code" now. One to open edit code prompt and second for confirming the edit. they somehow need to have different names.

Reply:
I missed your second comment about `...`.
In this case: `<override-text place="GoToAction" text="Cody: Edit Code"/>`
Still we need to add `Cody: ` part.

Reply:
done